src/css/wizard-navigation-bar.scss

```diff
@@ -0,0 +1,394 @@
+// Temporary variables used to pass parameters from the `define-style` mixin to its content block.
+// They will no longer be needed after content arguments support is released:
+//
+// - https://github.com/sass/sass/issues/871
+// - https://github.com/sass/libsass/releases
+// - https://github.com/sass/node-sass/releases/
+//
+// Using global variables is the recommended workaround
+//
+// - https://github.com/sass/sass/issues/626#issuecomment-345432420
+// - https://github.com/sass/sass/issues/871
+
+// indicator state: 'default' or 'hover'
+$wz-indicator-state_: null !global;
+
+// base color for one of possible indicator states (default, current, done etc.).
```

Comment:
I think you are redefining the indicator states compared to two lines above, i.e.: `default` and `hover` vs `default`, `current` and `done`

Reply:
Yes, "indicator state" has two meanings here (a naming issue). Perhaps we can call
- `default`, `hover` - indicator state
- `default`, `current`, `done` - wizard step state

?

Reply:
Sounds fine to me

Reply:
Fixed in 9593c6f